integration_tests/src/main/python/conftest.py

```diff
@@ -86,6 +87,16 @@ def is_utc():
 def is_not_utc():
     return not is_utc()
 
+def is_supported_time_zone():
+    """
+    Is current TZ supported, forward to Java TimeZoneDB to check
+    """
+    tz = get_test_tz()
+    jvm = spark_jvm()
+    ret = jvm.org.apache.spark.sql.rapids.TimeZoneDB.isSupportedTimeZone(tz)
```

Comment:
Could we memorize this? Calling back into the JVM can be expensive. And if this is never going to change I would much rather have us keep the result cached.

Reply:
Can we also use the `GpuTimeZoneDB` version of this?

Reply:
Now cached the support info; And updated to GpuTimeZoneDB.